core/src/scss/utilities/mixins/link/_link-icon.scss

```diff
@@ -0,0 +1,38 @@
+@charset "UTF-8";
+
+//
+// @link-icon($link-icon, $width: 0.65em, $animate: true)
+//
+// Display an icon after a link.
+//
+// $icon {string} - Basename of link icon .svg file (without the .svg suffix).
+// $width {string} - Width of icon including unit, e.g. 0.75em, 24px.
+// $animate {boolean} - Whether or not to animate on hover.
+//
+// Style guide: Mixins.Link.link-icon
+//
+@mixin link-icon($icon, $width: 0.65em, $animate: true) {
+  text-decoration: none;
+  @supports (mask-repeat: no-repeat) {
+    &::after {
+      @include margin(null null -1px 0.4em);
+      @include size($width);
+      display: inline-block;
+      content: '';
+      mask: url("#{$image-path}/#{$icon}.svg") no-repeat 0 0;
+      mask-size: contain;
+      background-color: color(link);
+      transition: margin-left 0.2s ease-in-out;
+    }
+
+    &:hover,
+    &:focus {
+      &::after {
+        background-color: color(link--hover);
+        @if $animate {
```

Comment:
I like the mixins, @JBCSU . I think we need more options for `$animate` other than true/false though. John mentioned that the external link icon would move towards the top-right direction for example, and I imagine the download one or the jump link icon would move downward when hovered

Reply:
I don't think that's a good idea. But if we do decide to do it, I propose we make a new ticket and do a separate PR. I don't believe that behavior was specified in the scope for the two tickets I'm addressing.

Reply:
Haha...I don't think having different animations is that bad. It would feel more natural for a down arrow to animate by moving down for example. I agree though that we can open a separate ticket for those.